examples/MvcWeb/TheraLang/Entities/Project.cs

```diff
@@ -7,5 +7,7 @@ public class Project
         public string Type { get; set; }
 
         public string Name { get; set; }
+        
+        public int StatusId { get; set;  }
```

Comment:
here instead of int use enum you created
```suggestion
        public ProjectStatus StatusId { get; set;  }
```